string yaml_file
string recording_id
---
string timestamp
bool suc
string message